ui/src/app/views/concept-set-details.tsx

```diff
@@ -57,15 +57,14 @@ const ConceptSetMenu: React.FunctionComponent<{
     content={ <React.Fragment>
       <TooltipTrigger content={<div>Requires Write Permission</div>}
                       disabled={canEdit}>
-        <MenuItem icon='pencil'
-                  onClick={() => onEdit}
+        <MenuItem onClick={() => onEdit}
                   disabled={!canEdit}>
           Edit
         </MenuItem>
       </TooltipTrigger>
       <TooltipTrigger content={<div>Requires Owner Permission</div>}
                       disabled={canDelete}>
-        <MenuItem icon='trash' onClick={onDelete} disabled={!canDelete}>
+        <MenuItem onClick={onDelete} disabled={!canDelete}>
```

Comment:
I made an assumption that all Menu dropdowns should have their icons removed. That being said, I'm not sure where this is and might need some help with finding it in the UI.

Reply:
this is in the concept set details page, there's a little ellipsis menu next to the concept set name where you can perform some actions on the set.

Reply:
thanks!